lib/Common/PlatformAgnostic/AssemblyCommon.h

```diff
@@ -0,0 +1,30 @@
+//-------------------------------------------------------------------------------------------------------
+// Copyright (C) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE.txt file in the project root for full license information.
+//-------------------------------------------------------------------------------------------------------
+#pragma once
+
+#ifndef _WIN32
+
+extern "C" void __register_frame(const void* ehframe);
+extern "C" void __deregister_frame(const void* ehframe);
+
+#if defined(_AMD64_) && !defined(DISABLE_JIT)
```

Comment:
I'm ok if you insist. Just want to note that another reason I prefer `ENABLE_NATIVE_CODEGEN` is it is positive. Easier to read and understand than double negative `!defined(DISABLE_JIT)`.